### PR TITLE
Initial import of Data Principles from google doc

### PR DESCRIPTION
**Pull request by alaric-dfe**
Source: https://docs.google.com/document/d/1GC8H1QgiVMJRbNhG562C-abdxNgUvHbwUN0E0r2Tfbw/edit

Formatted in Markdown, and slightly rephrased/structured to match other
principles pages within this repo.

### CODE REVIEW

File: source/principles/data/index.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+last_reviewed_on: 2022-01-24
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+## Context
+
+Our work is governed by the [DfE Digital Enterprise Data Architecture Principles](https://dfe-digital.github.io/architecture/principles/enterprise-data-architecture-principles/); this document explains how we apply them to our Data Infrastructure. Please read the DfE principles before continuing, as this document implements, not replaces, them!
```
Comment (luke-slowen) [inline source/principles/data/index.html.md.erb:12]: Something needed here to be explicit on the context in which these apply (i.e. digital services). It's implicit from it being in the tech guidance, but might need calling out?

**Comment by govtim** [comment]
I think the guidelines might be better split into a guide cross referencing to/from places like the API guidelines would be great I am just working on adding security and Operational Data Interchange Standards there

File: source/principles/data/index.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+We are bound by the following regulations, and you should make yourself familiar with them:
+
+- [GDPR](https://www.gov.uk/government/publications/guide-to-the-general-data-protection-regulation) (General Data Protection Regulation)
+- [Data Protection Act](https://www.gov.uk/government/publications/data-protection-act-2018-overview)
+- [Freedom of Information Act](https://www.legislation.gov.uk/ukpga/2000/36/contents)
+- [DfE Data Policies](https://educationgovuk.sharepoint.com/sites/lvewp00053/SitePages/Policies-and-Standards.aspx)
+- Service teams are responsible for performing the DPIA (Data Protection Impact Assessment) for sensitive data they collect, including its further storage and processing as part of larger projects such as service line analytics.
+- The Data Infrastructure team, however, are responsible for security of that data once it has entered cross-team infrastructure. Therefore, they must provide information to the service teams to enable them to correctly assess the data protection impact of handing data over.
+
+## Guidelines
```
Comment (luke-slowen) [inline source/principles/data/index.html.md.erb:78]: I think it's helpful to have guidance here, as a follow-on from the principles, but might need some clearer section segregation so folk reading know they're moving from a principle, to how to implement it. I contemplated whether this should be split out into the guidance section... but probably not.

Comment (luke-slowen) [reply source/principles/data/index.html.md.erb:78]: ah ha @govtim, similar thoughts

File: source/principles/data/index.html.md.erb (hunk 1 above)
Comment (alaric-dfe) [reply source/principles/data/index.html.md.erb:12]: Done

**Review by luke-slowen — APPROVED** [review]
Looks good, happy previous comments have been addressed, few more tweaks for readability in latest commit.

**Pull request merged**